src/views/LogoutSuccess.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+    <div class="success center">
+        <div class="icon-checkmark-circle"></div>
+        <h1>Logout<br>was successfull!</h1>
+        <div style="flex-grow: 1;"></div>
+        <button @click="close">Back to {{ request.appName }}</button>
+    </div>
+</template>
+
+<script lang="ts">
+import {Component, Emit, Vue} from 'vue-property-decorator';
+import {ParsedLogoutRequest} from '../lib/RequestTypes';
+import {State} from 'vuex-class';
+import {KeyInfo, KeyStorageType} from '../lib/KeyInfo';
+import {ResponseStatus, State as RpcState} from '@nimiq/rpc';
+import {RemoveKeyResult} from '@nimiq/keyguard-client';
+import { AddressInfo } from '@/lib/AddressInfo';
+import { KeyStore } from '@/lib/KeyStore';
+import { Static } from '@/lib/StaticStore';
+
+@Component({components: {}})
+export default class LogoutSuccess extends Vue {
+    @Static private request!: ParsedLogoutRequest;
+    @Static private rpcState!: RpcState;
+    @State private keyguardResult!: RemoveKeyResult;
+
+    public mounted() {
+        if (this.keyguardResult.success === true) {
+            KeyStore.Instance.remove(this.request.keyId);
+        }
+        this.rpcState.reply(ResponseStatus.OK, this.keyguardResult);
```

Comment:
The page should only return to the caller when the user clicks the button. The deletion of the database entry should happen immediately, but the `reply` only when the `Back to <appName>` button is clicked. Thus the reply should be moved into the `close()` method to replace the `window.close()`.